index.html

```diff
@@ -1,93 +1,85 @@
+{{define "index"}}
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
-  <title>Starter Template - Materialize</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
```

Comment:
I have no idea what the format of this should be. Is there some web/backend resource we can use from Intrepid?

Reply:
let me check sherpa

Reply:
not really, theres some best practices but nothing as formal as our style guide.